pagopa/pn-frontend-e2e
# Pn-9709-controllo-messaggio-errore-dopo-tre-OTP-errati

## Short description

New Test: try to insert PEC, email and cellular number and for each of these insert wrong OTP 3 times and check the error message

## List of changes proposed in this pull request

- new file feature for the test
- new general steps for the test
- new background method to check the correct OTP pop-up display, wrong OTP insertion, click on the confirm button 3 times and error message check

## How to test

Simply run `mvn clean test -D"cucumber.filter.tags=@TA_inserimentoTreOTPErratiPerTuttiIRecapiti"` and verify if the test works fine.
